content/docs/user-guide/managing-external-data.md

```diff
@@ -61,6 +61,11 @@ types:
    moves a local file to the external location. This produces an external output
    in `dvc.yaml`.
 
+> 🔒 Note that for locations that require authentication, it's necessary to
```

Comment:
this is not exactly correct, auth can be done via env vars, or regular credentials by the underlying library (like boto3).

Reply:
True. Updated, PTAL